code/common/clock.cc

```diff
@@ -55,37 +75,63 @@ namespace common::time
   {
     reset();
 
-    m_start_cycle = current_cycle_count();
+    #if defined(_WIN32)
+      m_start_cycle = current_cycle_count();
+    #elif defined(__linux__)
+      this->m_start_time = current_timestamp();
```

Comment:
No `this->` prefix please.

Reply:
Addressed in 95a120ab595e8373fb27e8519944e828bcb9d8fd